download_and_install/compiling_from_source.md

```diff
@@ -0,0 +1,242 @@
+        libncurses-dev \
+        libprotobuf-dev \
+        libprotoc-dev \
+        libtool \
+        libusb-1.0.0-dev \
+        make \
+        protobuf-compiler \
+        pkg-config \
+        uuid-dev \
+        zlib1g-dev
+
+CentOS/RHEL/Fedora
+------------------
+
+Distributions using RPM packages can install all dependencies with:
+
+    sudo dnf install \
+        autoconf \
+        automake \
+        avahi-devel \
+        bison \
+        cppunit-devel \
+        flex \
+        gcc-c++ \
+        libftdi-devel \
+        liblo-devel \
+        libmicrohttpd-devel \
+        libtool \
+        libusbx-devel \
+        make \
+        ncurses-devel \
+        protobuf-lite-devel \
+        uuid-devel \
+        zlib-devel
+
+macOS
+-----
+
+Dependencies are available from [MacPorts](https://www.macports.org/):
+
+    sudo port install \
+        autoconf \
+        automake \
+        bison \
+        cppunit \
+        flex \
+        libftdi1 \
+        liblo \
+        libmicrohttpd \
+        libtool \
+        libusb \
+        libuuid \
+        ncurses \
+        pkgconfig \
+        protobuf3-cpp \
+        zlib
+
+or [Homebrew](https://brew.sh/):
+
+    brew install \
+        autoconf \
+        automake \
+        bison \
+        cppunit \
+        flex \
+        libftdi \
+        liblo \
+        libmicrohttpd \
+        libtool \
+        libusb \
+        libuuid \
+        ncurses \
+        pkg-config \
+        protobuf \
+        zlib
+
+Python dependencies
+-------------------
+
+If you wish to build the Python library, additional libraries are required:
+
+    pip3 install --user \
+        numpy \
+        protobuf
+
+If the pip3 command does not exist, install python3-pip from your package manager and try again.
+
+Build
+=====
+
+If this is the first time compiling with this download, generate the build system:
+
+    autoreconf -i
+
+OLA uses autotools for building. Run
+
+     ./configure --help
+
+to see all options. The most popular option is `--enable-python-libs` to build the Python client module. If you want to
+use the RDM responder tests, add `--enable-rdm-tests`.
+
+Once you’ve decided on the options, it’s time to build OLA.
+
+    ./configure <your options>
+    make -j$(nproc)
+    make check
+    sudo make install
+
+Mac users should run use `make -j$(sysctl -n hw.logicalcpu)` instead of `make -j$(nproc)`.
+
+Finally, run `sudo ldconfig` to make new libraries available.
+
+Device drivers
+==============
+
+Note that, for some devices, it is necessary to install drivers for OLA to work with them. For example, the
+[Open DMX USB](http://opendmx.net/index.php/Open_DMX_USB "Open DMX USB") device needs an additional kernel module that
+could be built using the instructions on
+[LLA\_and\_Q\_Light\_Controller\_Ubuntu\_Tutorial](http://opendmx.net/index.php/LLA_and_Q_Light_Controller_Ubuntu_Tutorial)
+. For other devices, refer to the corresponding device page on the wiki.
```

Comment:
Nowadays using the FTDI plugin is probably easier.

Reply:
This section is also copied verbatim from the old text.  I've never played with FTDI functionality, so may need to do some research here.  I'm all for removing unnecessary references to the often-outdated wiki.

Reply:
See https://www.openlighting.org/ola/get-help/ola-faq/#What_is_the_difference_between_the_different_USB_plugins

I suspect the kernel plugin would in theory be more efficient, but for most people it's probably not worth the hassle of getting that going to.